styles/styles.css

```diff
@@ -0,0 +1,807 @@
+    font-size: 1rem;
+    font-weight: 600;
+    color: var(--color-darkOrange);
+  }
+
+  .speaker-infos .separator {
+    background: var(--color-dark);
+    margin: 0.3em 0 0.6em 0;
+  }
+
+  .speaker-infos .description {
+    font-size: 1rem;
+  }
+
+  .btn-more {
+    width: 97%;
+    align-items: center;
+    gap: 1em;
+    text-transform: uppercase;
+    color: #000;
+    background: var(--color-white);
+  }
+
+  .btn-more .downArrow {
+    width: 6px;
+    height: 6px;
+    border-bottom: 2px solid var(--color-darkOrange);
+    border-right: 2px solid var(--color-darkOrange);
+    transform: rotate(45deg);
+    padding: 0.2em;
+    margin-top: -0.5em;
+  }
+
+  .btn-more.active .downArrow {
+    transform: rotate(230deg);
+    margin-top: -0.2em;
+  }
+
+  /* ==== PARTNER SECTION ==== */
+  .partner-section {
+    color: #d3d3d3ba;
+    background: var(--color-dark);
+    padding-top: 1em;
+    display: none;
+  }
+
+  .about-page.partner-section {
+    display: block;
+  }
+
+  .partner-box {
+    max-width: max-content;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: flex-end;
+    gap: 1.5em;
+    margin: 0 auto;
+    flex-wrap: wrap;
+    font-weight: 600;
+    font-size: 1.45rem;
+  }
+
+  /* ==== ABOUT LEADER WEEK SECTION ==== */
+  .about-leader-week {
+    text-align: center;
+    padding: 1em 0.5em;
+    border-top: 1px solid var(--color-grey);
+  }
+
+  .about-leader-week img {
+    max-width: 250px;
+    border: 1px solid var(--color-grey);
+    margin: 1em 0;
+  }
+
+  /* ==== LATEST EVENTS SECTION ==== */
+  .latest-events {
+    text-align: center;
+    padding: 1em 0.5em;
+    border-top: 1px solid var(--color-grey);
+  }
+
+  .events-box-container {
+    flex-direction: column;
+  }
+
+  .events-box {
+    width: 80%;
+    max-width: 350px;
+    height: 200px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin: 1em auto;
+    padding: 0 0.5em;
+    color: var(--color-white);
+    position: relative;
+  }
+
+  .events-box::after {
+    position: absolute;
+    content: '';
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: var(--color-darkOrange);
+    opacity: 0.6;
+  }
+
+  .events-box span {
+    font-size: 2rem;
+    font-weight: 600;
+    z-index: 1;
+  }
+
+  .events-box p {
+    font-size: 1.1rem;
+    z-index: 1;
+    opacity: 0.9;
+  }
+
+  .events-box.box-1 {
+    background: url(../images/event-1.jpg) center/cover no-repeat;
+  }
+
+  .events-box.box-2 {
+    background: url(../images/event-2.jpg) center/cover no-repeat;
+  }
+
+  /* ==== FOOTER ==== */
+  footer {
+    justify-content: space-evenly;
+    align-items: center;
+    gap: 1.5em;
+    padding: 2em 0;
+    border-top: 1px solid var(--color-grey);
+  }
+
+  .footer-2 {
+    display: none;
+  }
+}
+
+/* ==== Mobile M - 375px ==== */
+@media screen and (width >= 375px) {
+  .section-container {
+    padding: 2em 1em;
+  }
```

Comment:
- [ ] There seems to be a text overflow in the landing page of both the Home and About pages and this is making your UI look unprofessional so I highly advise you to work on them. 
- Here's how it looks:
  ![image](https://github.com/lily4178993/leaders-week/assets/104968236/c764ecd7-5817-4e0f-b170-01f1d4ba64d9)
  ![image](https://github.com/lily4178993/leaders-week/assets/104968236/19db0d41-2435-46a3-8bbf-a01145cb840f)